Serialiser_Engine/Convert/Json.cs

```diff
@@ -98,6 +96,44 @@ public static object FromJson(string json)
         }
 
         /*******************************************/
+
+        [Description("Convert a List of objects to a Json array.")]
+        [Input("obj", "Object to be converted.")]
+        [Output("json", "String representing the object in json.")]
+        public static string ToJsonArray(this IEnumerable<object> objs)
+        {
+            List<string> allLines = new List<string>();
+            string json = "";
+
+            // Join all individually serialised objects with a comma and a newline
+            json = string.Join(",\n", objs.Where(c => c != null).Select(obj => obj.ToJson()));
+
+            // Put between square brackets, to make a valid JSON array.
+            json = "[" + json + "]";
+
+            return json;
+        }
+
+        /*******************************************/
+
+
+        [Description("Convert a Json string to a BHoMObject")]
```

Comment:
This description doesn't seem to match the function? Should it be:
```suggestion
        [Description("Convert a Json string to a collection of objects")]
```

Name as it's returning an IEnumerable, and of object rather than BHoMObject?